src/pages/Admin/Venue/VenueDetailsForm.tsx

```diff
@@ -0,0 +1,351 @@
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from "react";
+import { useForm } from "react-hook-form";
+import { useHistory } from "react-router-dom";
+import Bugsnag from "@bugsnag/js";
+import * as Yup from "yup";
+
+import { IS_BURN } from "secrets";
+
+import {
+  DEFAULT_VENUE_BANNER,
+  DEFAULT_VENUE_LOGO,
+  PLAYA_HEIGHT,
+  PLAYA_IMAGE,
+  PLAYA_VENUE_NAME,
+  PLAYA_VENUE_SIZE,
+  PLAYA_VENUE_STYLES,
+  PLAYA_WIDTH,
+} from "settings";
+
+import {
+  createUrlSafeName,
+  createVenue,
+  updateVenue,
+  VenueInput,
+} from "api/admin";
+
+import { setSovereignVenue } from "store/actions/SovereignVenue";
+
+import { UserStatus } from "types/User";
+import { ExtractProps } from "types/utility";
+import { VenuePlacementState } from "types/venues";
+
+import { isTruthy } from "utils/types";
+
+import { useDispatch } from "hooks/useDispatch";
+import { useQuery } from "hooks/useQuery";
+import { useSovereignVenue } from "hooks/useSovereignVenue";
+import { useUser } from "hooks/useUser";
+
+import { PlayaContainer } from "pages/Account/Venue/VenueMapEdition";
+import { VenueDetailsSubForm } from "pages/Admin/Venue/VenueDetailsSubForm";
+
+import "firebase/functions";
+
+import {
+  editVenueCastSchema,
+  validationSchema,
+} from "./DetailsValidationSchema";
+import { WizardPage } from "./VenueWizard";
+
+// @debt refactor any needed styles out of this file (eg. toggles, etc) and into DetailsForm.scss/similar, then remove this import
+import "../Admin.scss";
+import "./Venue.scss";
+
+export type FormValues = Partial<Yup.InferType<typeof validationSchema>>; // bad typing. If not partial, react-hook-forms should force defaultValues to conform to FormInputs but it doesn't
```

Comment:
```suggestion
export type FormValues = Partial<Yup.InferType<typeof validationSchema>>; // bad typing. If not partial, react-hook-forms should force defaultValues to confirm to FormInputs but it doesn't
```

Reply:
I don't think that's a typo. https://www.merriam-webster.com/dictionary/conform